manual_flagging/flagging.py

```diff
@@ -21,17 +21,25 @@
     inputs = yaml.safe_load(stream)
 
 # Validate the input specification
+
 # Check housing_market_type
-assert "housing_market_type" in inputs, "Missing key: 'housing_market_type'"
 assert set(inputs["housing_market_type"]).issubset(
-    {"res_single_family", "res_multi_family", "condos"}
-), "housing_market_type can only contain 'res_single_family', 'res_multi_family', 'condos'"
+    {"res_single_family", "res_multi_family", "res_all", "condos"}
+), "housing_market_type can only contain 'res_single_family', 'res_multi_family', 'res_all', 'condos'"
 assert len(inputs["housing_market_type"]) == len(
     set(inputs["housing_market_type"])
 ), "Duplicate values in 'housing_market_type'"
 
+# Check housing_market_type for mutual exclusivity
+assert not (
+    (
+        "res_single_family" in inputs["housing_market_type"]
+        or "res_multi_family" in inputs["housing_market_type"]
+    )
+    and "res_all" in inputs["housing_market_type"]
+), "Either 'res_single_family' and 'res_multi_family' can be included, or 'res_all', but not both"
+
```

Comment:
Check to make sure a subsection of `"res_all"` isn't selected along iwth `"res_all"`.

Reply:
Scrapped this change as I'm not sure it will be needed.